source/initial-dns-seedlist-discovery/tests/README.md

```diff
@@ -3,6 +3,13 @@
 This directory contains platform-independent tests that drivers can use to prove their conformance to the Initial DNS
 Seedlist Discovery spec.
 
+### Prose Tests
+
+When given a valid SRV with
+
+1. Only one domain level (ex: `{localhost}`)
+2. Only two domain levels (ex: `{mongo.local}`) Parse the connection string without throwing a parse related error.
```

Comment:
EOD, will be OOO for aug 26-30,  but note for self when I return: need to add testing for stricter parent match as well.